### DISCUSSION TITLE
using the meta data in the template but it can't work

### DISCUSSION THREAD

**Discussion by fred1357944** [BryanJenksCommunity/FAQ#85] [Q&A] [answered]
<img width="694" alt="image" src="https://user-images.githubusercontent.com/34299763/124428900-ccc60480-dd9f-11eb-8e8a-1b6ce4c63e3f.png">

when I import the meta template into my new file named "{test" but it shows up this announcement. Does anyone know how to fix this?

Thank you so much...

**Comment by tallguyjenks** [comment]
can you show me screen shots of the following:

- the command palette command you're running to insert the template
- the template itself with the templater code
- your templates folder and the files within it

**Comment by fred1357944** [comment]
Thank you so much.

1-3.The Templator settings
4. My new file name and file location
5. Importing commend palette
6. The meta file

Best
Fred

**Reply by tallguyjenks** [reply]
no screenshots provided?

**Reply by fred1357944** [reply]
Sorry the attachment was in the mail… didn’t show up here

**Comment by fred1357944** [comment]
![911EB4C5-5087-421D-B0A5-ACBB2FA00A83](https://user-images.githubusercontent.com/34299763/125481223-3bc17dec-68a6-4404-8f79-98c4ea860ed5.png)
![B58BC74B-BF45-442C-B720-127E3A83E938](https://user-images.githubusercontent.com/34299763/125481232-fcd78203-b81c-484a-b79e-0a592ed5fba7.png)
![388BAC99-B4DC-42A5-8857-F91C775B9669](https://user-images.githubusercontent.com/34299763/125481239-ed837a6d-ae1c-47f3-8587-f348f1a4a5f7.png)
![7A41F4A5-7EAC-43E4-A8D2-5BAD4F730DB6](https://user-images.githubusercontent.com/34299763/125481242-7110240b-579f-4cf8-9715-0bde6f43217c.png)
![BB678FAC-C5A0-44C0-9352-1B269B13BFB4](https://user-images.githubusercontent.com/34299763/125481249-d4a631ca-5c5c-4f43-934a-b46819a08d78.png)

**Reply by tallguyjenks** [reply]
okay, it looks like its all setup just fine, just make sure the command youre using to insert templates isnt the built in template command but rather the templater command to insert tempalte

**Reply by fred1357944** [reply]
Thanks! I will try today :)

**Comment by julian2405** [comment]
I got the same settings as above and use the Templaer command but I still get the same error

**Reply by D7891** [reply]
I have the same issue....🤓

**Comment by tallguyjenks** [comment, marked as answer]
@fred1357944 @julian2405 @D7891 i think i got it, in the image here:

![image](https://user-images.githubusercontent.com/29872822/128650281-2168fbf6-888c-40aa-bc12-8170d678d869.png)

the links to the templates should say instead of 
`"Templates/Inputs/Tweet"` 
it should be
`"[[Tweet]]"`

change those settings and see if they work

**Reply by fred1357944** [reply]
still can't I am sorry

**Reply by D7891** [reply]
it doesn't work by me either....sorry also....thank you for answering Bryan

**Reply by tallguyjenks** [reply]
this is so strange. Not sure why this is not working for you all.

This is my meta template:

```
<%* if (tp.file.title.charAt(0) == "{") { %>
<%-tp.file.include("[[Book]]")%>
<%* } else if (tp.file.title.charAt(0) == "@") { %>
<%-tp.file.include("[[Person]]")%>
<%* } else if (tp.file.title.charAt(0) == "!") { %>
<%-tp.file.include("[[Tweet]]")%>
<%* } else if (tp.file.title.charAt(0) == "%") { %>
<%-tp.file.include("[[Podcast]]")%>
<%* } else if (tp.file.title.charAt(0) == "+") { %>
<%-tp.file.include("[[Youtube]]")%>
<%* } else if (tp.file.title.charAt(0) == "(") { %>
<%-tp.file.include("[[Article]]")%>
<%* } else if (tp.file.title.charAt(0) == "&") { %>
<%-tp.file.include("[[Paper]]")%>
<%* } else if (tp.file.title.charAt(0) == "=") { %>
<%-tp.file.include("[[Thought]]")%>
<%* } else { %>
<%-tp.file.include("[[New]]")%>
<%* } _%>
```

inserting a template uses the template command not the core template plugin

my templates are all in a folder called "Templates"

.___.

**Reply by fred1357944** [reply]
It worked! I tried to disable the core template plugin and it successed!

**Reply by fred1357944** [reply]
Thanks a lot again!!

**Reply by D7891** [reply]
THANK YOU!!! It works! 🙏

**Reply by codefr3sh** [reply]
Thank you, this worked.

**Reply by moorebi** [reply]
Pulled this in and replaced the one in your "vault.zip" file with this one and works perfectly .. 
Thanks

**Reply by tallguyjenks** [reply]
Ive also updated the vault, someone pointed out that the old version was still in there so hopefully no one needs this answer anymore 😅

**Reply by moorebi** [reply]
Hoe can someone get the updated vault from you instead of going back and signing up for newsletter?

**Reply by tallguyjenks** [reply]
@moorebi https://github.com/BryanJenksCommunity/FAQ/discussions/108
